netket/experimental/operator/fermion.py

```diff
@@ -32,7 +32,11 @@ def destroy(
     Args:
         hilbert: The hilbert space.
         site: the site on which this operator acts.
-        sz: spin projection quantum number (e.g. sz=-0.5 for a spin-1/2 down)
+        sz: spin projection quantum number. This is the eigenvalue of
```

Comment:
maybe we should comment that we use the eigenvalues of the pauli matrices, still I am not entirely sure it's nice to do this change... (I understand it's consistent with Spin, but not super nice)